py/modthread.c

```diff
@@ -175,6 +175,7 @@ STATIC void *thread_entry(void *args_in) {
     ts.gc_lock_depth = 0;
 
     ts.mp_pending_exception = MP_OBJ_NULL;
+    ts.nlr_jump_callback_top = NULL;
```

Comment:
This should go before `mp_pending_exception`, to match the order in the state struct definition.

Reply:
Done